package.json

```diff
@@ -1,23 +1,25 @@
 {
   "name": "firestorm-db",
-  "version": "1.13.0",
+  "version": "2.0.0",
   "description": "Self hosted Firestore-like database with API endpoints based on micro bulk operations",
-  "main": "src/index.js",
-  "types": "typings/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "type": "module",
   "scripts": {
+    "build": "tsc",
     "php:start": "node tests/php_setup.mjs",
     "php:stop": "sh tests/php_server_kill.sh",
-    "php:tests": "pnpm php:stop ; pnpm php:start && pnpm js:tests ; pnpm php:stop",
-    "js:tests": "mocha tests/**/tests-setup.spec.mjs tests/**/*.spec.mjs",
-    "docker:build": "bash docker/docker_image_build.bash",
+    "php:tests": "pnpm build ; pnpm php:stop ; pnpm php:start && pnpm js:tests ; pnpm php:stop",
```

Comment:
so maybe find something better to get return value, a bash script?

Reply:
Fixed!